src/univariate/continuous/kolmogorov.jl

```diff
@@ -145,12 +145,12 @@ function rand(d::Kolmogorov)
 end
 
 # equivalent to
-# rand(Truncated(Gamma(1.5,1.0),tp,Inf))
+# rand(Truncated(Gamma(1.5,1),tp,Inf))
 function rand_trunc_gamma()
-    tp = 2.193245422464302 #pi^2/(8*t^2)    
+    tp = 2.193245422464302 #pi^2/(8*t^2)
```

Comment:
Correct. A few distributions (ksdist, kolmogorov, and maybe one or two others) didn't have any parameters to make parametric.